### ISSUE TITLE
Intent to archive

### ISSUE THREAD

**Issue by simoncozens** [googlefonts/nototools#634]
This repository contains a lot of tools which were once used in the Noto production process; however, the process has changed and simplified over time, and the only parts of the repo which are now used (by the Noto team) are `noto_lint.py` and `test_vertical_extents.py`.

`noto_lint.py` should really be replaced by the Noto profile in fontbakery; `test_vertical_extents.py` is a handy script which could find a home in gftools.

Having the other tools around is misleading, as it implies they are used by Noto when they aren't. However, if there are tools out there which people are using, I'd like to hear about it, so we can think how those uses can be best supported. If I don't hear anything, I'm planning to convert two tools we do use and then the archive this repo by the end of April.

**Comment by rsheeter** [comment]
I believe some parts of this are actively used by https://github.com/googlefonts/noto-emoji and therefore we should not deprecate. Pruning away long-unused parts is fine, maybe we can rescope to that?

**Comment by davelab6** [comment]
Moving noto-emoji specific parts to noto-emoji sounds good to me.

Youseedee also sounds good to me, if you add an Apache license and the other [typical G-OSPO stuff](https://opensource.google/documentation/reference/releasing/preparing) and transfer to me then I will transfer to /googlefonts and you can take it from there?

**Comment by davelab6** [comment]
I defer to Rod on these choices :)

**Comment by anthrotype** [comment]
(technically I am off until next week - just wanted to point out unicodedata2 is simply a backport of the latest cpython unicodedata and as such it is bound by the upstream)

**Comment by simoncozens** [comment]
Yeah, youseedee and unicode_data both provide a much richer interface to the whole of the UCD, not just the limited bits that the Python API exposes. unicodedata2 would need a lot of work to come up to the same level.

**Comment by marekjez86** [comment]
@simoncozens : please add me to PRs

**Comment by simoncozens** [comment]
@rsheeter @marekjez86 @anthrotype please see

https://github.com/googlefonts/nototools/pull/637
https://github.com/googlefonts/nototools/pull/638
https://github.com/googlefonts/nototools/pull/639
https://github.com/googlefonts/nototools/pull/640
https://github.com/googlefonts/nototools/pull/641

(for starters)

**Comment by rsheeter** [comment]
Merged the first three, defer to @marekjez86 on #640, approved #641 but did not merge as it appeared to be reporting anger from CI.

Lets keep unicode_data here for now.
